components/zcash_address/src/kind/unified.rs

```diff
@@ -261,27 +268,27 @@ pub trait FromReceivers: private::SealedContainer + std::marker::Sized {
             Err(ParseError::OnlyTransparent)
         } else {
             // All checks pass!
-            Ok(Self::from_inner(receivers))
+            Ok(Self::from_inner(items))
         }
     }
 }
 
 /// Trait providing common decoding logic for Unified containers.
```

Comment:
This comment is out of date, and my prior suggestion stands:
```suggestion
/// Trait for for Unified containers, that exposes the items within them.
```